tests/units/inventory_models_test.py

```diff
@@ -61,7 +62,6 @@ def test_anta_inventory_host_invalid(self, test_definition):
             host_inventory = AntaInventoryHost(host= test_definition['input'])
```

Comment:
This could be written
```
with pytest.raises(ValidationError):
  host_inventory = AntaInventoryHost(host= test_definition['input'])
```

If no exception is raised, the test will fail